troubleshooting/known_issues/cfs-api_pods_in_CLBO_state.md

```diff
@@ -0,0 +1,40 @@
+# `CFS-API` pods in CLBO state during CSM install
+
+## Issue Description
+
+When installing CSM 1.6, `cray-shared-kafka-kafka-*` pods in services namespace fail to come up which would result in `CFS-API` pods in CLBO state. This happens because of an issue with Zookeeper related to slow DNS.
```

Comment:
```suggestion
When installing CSM 1.6, `cray-shared-kafka-kafka-*` pods in the services namespace fail to come up which results in `CFS-API` pods in CLBO state. This happens because of an issue with Zookeeper related to slow DNS.
```